misc/bootstrap-overrides.css

```diff
@@ -0,0 +1,7 @@
+/* This is needed since ngDocs has bootstrap as a default file to load
+ * patternfly assumes that the consuming application is not loading the bootstrap.css directly
+  */
```

Comment:
Sure, in ngDocs index.tmpl they have the following line: addTag('link', {rel: 'stylesheet', href: 'css/bootstrap.min.css', type: 'text/css'});

This pulls in the default out of box bootstrap.css. The patternfly.css is meant to be installed as a replacement for this file not as a stand aside. So this was causing conflicts with the form grids. If you remove that style you will see on all the examples that use a 12 column grid the labels and fields are not on the same line. This fixes that issue by addressing the style clash.